libgalois/include/katana/PropertyGraph.h

```diff
@@ -237,6 +237,43 @@ class KATANA_EXPORT PropertyGraph {
     rdg_.set_view_name(fmt::format("rdg-{}-part{}", part_policy, num_hosts));
   }
 
+  const std::unordered_map<tsuba::EntityTypeID, std::vector<tsuba::EntityTypeID>>& node_entity_type_id_dictionary()
+    const {
+      return rdg_.node_entity_type_id_dictionary();
+  }
+  
+  void set_node_entity_type_id_dictionary(std::unordered_map<EntityTypeID, std::vector<EntityTypeID>>&&
```

Comment:
I have special hatred for setters. IMO, every setter could have a better name. In this case, it seems the true intention is something like `persist_node_entity_type...`.